src/main/java/uk/gov/hmcts/reform/civil/service/docmosis/dq/DirectionsQuestionnaireGenerator.java

```diff
@@ -207,6 +247,14 @@ private Boolean isRespondentState(CaseData caseData) {
 
     private Party getApplicant(CaseData caseData) {
         var applicant = caseData.getApplicant1();
+
+        if (TWO_V_ONE.equals(MultiPartyScenario.getMultiPartyScenario(caseData))
```

Comment:
the three can be combined within one if condition

Reply:
Done